libraries/IiifItems/IiifUtil.php

```diff
@@ -72,16 +72,21 @@ protected static function addDublinCoreMetadata(&$jsonData, $record) {
                 $jsonData['license'] = join('<br>', $elements['Dublin Core']['Rights']);
                 unset($elements['Dublin Core']['Rights']);
             }
-            if (!empty($elements['Dublin Core'])) {
-                if (!isset($jsonData['metadata'])) {
-                    $jsonData['metadata'] = array();
-                }
-                foreach ($elements['Dublin Core'] as $elementName => $elementContent) {
-                    $jsonData['metadata'][] = array(
-                        'label' => $elementName,
-                        'value' => join('<br>', $elementContent)
-                    );
-                }
+        }
+        if (!isset($jsonData['metadata'])) {
+            $jsonData['metadata'] = array(
+                array(
+                    'label' => __('Record in Omeka'),
+                    'value' => '<a href="' . record_url($record) . '">View page</a>'
+                )
+            );
+        }
+        foreach ($elements as $elementSetName => $elementSet) {
+            foreach ($elementSet as $elementName => $elementContent) {
+                $jsonData['metadata'][] = array(
+                    'label' => $elementName,
+                    'value' => join('<br/>', $elementContent)
```

Comment:
Kudos for using the XHTML form of `<br/>` as required by the IIIF spec when [HTML is used in property values](https://iiif.io/api/presentation/2.1/#html-markup-in-property-values). It's interesting that the spec seems to suggest that property values should be enclosed in start/end tags so that consuming applications can easily test if the value is plain text or HTML (e.g. `'<span>...</span>'`). I'm not sure I'd recommend doing that here if Mirador is parsing things OK as-is. But you might consider using `trim()` on the joined string to ensure there's no leading or trailing whitespace.